source/served/commands/complete.d

```diff
@@ -1053,7 +1053,7 @@ auto convertDCDIdentifiers(DCDIdentifier[] identifiers, bool argumentSnippets, D
 				// enum definitions are the enum identifiers (not the type)
 				detailDescription = "enum";
 			}
-			else if (identifier.type == "f" && dcdext)
+			else if ((identifier.type == "f" || (identifier.type == "l" || identifier.definition.indexOf(" ") != -1)) && dcdext)
```

Comment:
instead of checking for a space it would be better letting `extractCallParameters` do this (which it does by returning CalltipsSupport.init for syntax errors / unparsable functions)